transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_account.sql

```diff
@@ -1,38 +1,82 @@
-{{config({
-    "schema": "hightouch",
+{{ config(
+    { "schema": "hightouch",
```

Comment:
@munish7771 , can you check with @sri-byte  and @ifoukarakis  if this formatting is ok and is aligned with their owns? Otherwise, check-ins from different people on the same file will repeatedly change the formatting, making the PR reviews difficult to follow and prone to errors.

Reply:
Let's sync on the formatting.